processors/config/fspTuple_cfg.py

```diff
@@ -0,0 +1,206 @@
+import HpstrConf
```

Comment:
So my thinking was more this cfg would just do FSPs (many analyses can be done on just fsp), and we add FSPs to recoTuple_cfg.py